### ISSUE TITLE
Error b-form-group when using with Nuxt

### ISSUE THREAD

**Issue by BruceHem** [bootstrap-vue/bootstrap-vue#1301]
I'm using boostrap-vue with Nuxt.

And when I add b-form-group in, I get the following error.
![screen shot 2017-11-07 at 4 57 26 pm](https://user-images.githubusercontent.com/15667259/32487655-e101ed00-c3dc-11e7-8fc8-4a3ec53df83b.png)

I have tried remove tag by tag, this one seems to be the problem.

```pug
  .container
    b-card(
      title="Sign In"
      sub-title="Please, enter your sign in credentials"
    )
      b-form.card-text(@submit="submit")
        b-form-group#email-group(
          label="Email Address"
          label-for="email"
        )
          b-form-input#email(
            name="email"
            v-model="email"
          )
        b-form-group#password-group(
          label="Password"
          label-for="password"
        )
          b-form-input#password(
            name="password"
            v-model="password"
            type="password"
          )
        b-button(type="submit" variant="primary") Sign In
        b-button(variant="outline-secondary") Forgot Password?
```

**Comment by tmorehouse** [comment]
Related to issue #1296 

If you change your import of bootstrap-vue to:

```js
import Vue from 'vue'
import BootstrapVue from 'bootstrap-vue/dist/bootstrap-vue.esm'

Vue.use(BootstrapVue)
```

The issue should be fixed.

We are looking into why this is happening with the new `es` builds.

**Comment by pi0** [comment]
Thanks for the reports. It should be fixed in v1.0.1 now :)
